assets/sources/source_clients/amazon.py

```diff
@@ -43,9 +43,9 @@ def get_photo(self):
 class AmazonUsed(Amazon):
     """ A (hopefully) re-usable Amazon web-scraping client for used content."""
 
-    source = "Amazon"
     use_status = "Used"
 
     def get_price(self):
         # usedBuySection only works with New products
-        return self.soup.find(id="usedBuySection").text.replace("Buy used:", "").strip()
```

Comment:
This ID was nowhere to be found on the surface product page, right now the surface is the only product that uses AmazonUsed so this change won't break another product.